PS3 Toolbox/PS3RPC.cs

```diff
@@ -152,7 +152,7 @@ class PS3RPC {
                     return true;
                 }
 
-                PS3.SetMemory(RPC_BASE, new byte[0x1B0]);
+                PS3.SetMemory(RPC_BASE, new byte[0x100]);
```

Comment:
Even if this SetMemory is reduced, doesn't it get overwritten by the RPC instructions anyways? the RPC instructions are longer than 0x100 bytes

Reply:
Actually, read the initial comment again. I see what would work. Can you replace the size to match the size of the RPC Instructions rather than using 0x100 arbitrarily? `sizeof(RPC_INSTRUCTIONS)` something like that. I'll merge then